OpenRA.Mods.Common/Scripting/Global/CoordinateGlobals.cs

```diff
@@ -59,15 +59,15 @@ public class WVecGlobal : ScriptGlobal
 		public WVec Zero { get { return WVec.Zero; } }
 	}
 
-	[ScriptGlobal("WRange")]
-	public class WRangeGlobal : ScriptGlobal
+	[ScriptGlobal("WDist")]
```

Comment:
This is going to break scripts.  Please keep both, and have WRange log deprecation notices to the game chat to encourage the map author to change it during the next release cycle.